src/README.md

```diff
@@ -16,10 +16,13 @@ To build protobuf from source, the following tools are needed:
   * automake
   * libtool
   * curl (used to download gmock)
+  * make
+  * clang (gcc)
```

Comment:
You are right, clang is more my favourite, than general used compiler. I change it.